grr/server/grr_response_server/gui/ui/store/client_page_facade.ts

```diff
@@ -32,6 +33,17 @@ export type StartFlowState = {
   readonly error: string,
 };
 
+export type ChangeRequestState = {
```

Comment:
This can be simplified and `clearRemoveClientLabelState` removed:

We are only interested in the fact that a label was deleted successfully (to show the snackbar and remove it from the label list). You can introduce an observable that only emits the label name whenever the deletion was successful.

Reply:
This is a very good idea, thanks for the suggestion!